fedimint-client/src/lib.rs

```diff
@@ -1757,17 +1767,23 @@ pub struct ClientBuilder {
     primary_module_instance: Option<ModuleInstanceId>,
     admin_creds: Option<AdminCreds>,
     db_no_decoders: Database,
+    // unique Arc unless from_existing is called.
+    meta_service: Arc<MetaService>,
```

Comment:
Why would a builder have meta service, when it's unknown what the config is going to be an Federation. Seems to me like this should be create during actually `build`, when the config is known etc.

Reply:
so you can add source before building, we could have `MetaServiceBuilder` but that doesn't work with `ClientBuilder::from_existing`